### PR TITLE
update the gitpod setup description to be more precise.

### PR DESCRIPTION
**Pull request by nisarhassan12**
### Checklist
- [ ] Use `develop` as the base branch
- [x] Exclude the gulp build (`/dist` changes) from the PR
- [ ] Test on [supported browsers](https://github.com/sampotts/plyr#browser-support)

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -29,9 +29,15 @@ Please follow the instructions in our issue templates. Don't use github issues t
...
+### Online one-click setup for contributing
 
-Alternatively can also use Gitpod, a free online Visual Studio Code-like IDE. With a single click it will automatically launch a ready-to-code workspace with all the dependencies pre-installed, gulp watching for changes and the web server running, so that you can start coding straightaway.
+You can use Gitpod(a free online VS Code-like IDE) for contributing. With a single click it will launch a workspace and automatically:
+
+- clone the plyr repo.
+- install the dependencies.
+- run `gulp` to the start the server.
+
+so that you can start straight away.
```
Comment (sampotts) [inline contributing.md]: This should start with a capital shouldn't it?

File: contributing.md [hunk 2]
```diff
@@ -29,9 +29,15 @@ Please follow the instructions in our issue templates. Don't use github issues t
 
 * To test locally, you can use the demo site. First make sure you have installed the dependencies with `npm install` or `yarn`. Run `gulp` to build and it will run a local web server for development and watch for any changes.
 
-### Online one-click setup
+### Online one-click setup for contributing
 
-Alternatively can also use Gitpod, a free online Visual Studio Code-like IDE. With a single click it will automatically launch a ready-to-code workspace with all the dependencies pre-installed, gulp watching for changes and the web server running, so that you can start coding straightaway.
+You can use Gitpod(a free online VS Code-like IDE) for contributing. With a single click it will launch a workspace and automatically:
```
Comment (sampotts) [inline contributing.md]: I think we're missing a space before the parenthesis here

**Comment by nisarhassan12** [comment]
Thanks! I have made the changes. I'm not good at English.

**Comment by sampotts** [comment]
> Thanks! I have made the changes. I'm not good at English.

No problem at all! Thanks a lot for your help 👍

**Pull request merged**